src/app/components/card/card.component.less

```diff
@@ -0,0 +1,64 @@
+@import (reference) './../../../assets/stylesheets/_base';
+@import (reference) './../../../assets/stylesheets/shared/_layout';
+
+.board-card {
+  width: 100%;
+  float: left;
+  padding: 15px;
+  margin: 0 0 5px 0;
+  background: @color-pf-white;
+  .Border(2px, solid, @color-pf-black-400);
+  &.active {
+    border-color: @color-pf-blue-400;
+  }
+  >div {
+    p {
+      margin: 0;
+    }
+    .wi-type-icon {
+      padding: 0 6px; // padding-top,bottom:0 required here
+      font-size: 1px;
```

Comment:
isn't it too small?  this is the reason we are not able to see the work item type icon.